src/core/server/saved_objects/routes/create.ts

```diff
@@ -44,16 +44,16 @@ export const registerCreateRoute = (router: IRouter) => {
               })
             )
           ),
-          namespaces: schema.maybe(schema.arrayOf(schema.string(), { minSize: 1 })),
+          initialNamespaces: schema.maybe(schema.arrayOf(schema.string(), { minSize: 1 })),
```

Comment:
If beats is sending `namespaces` in it's payload, won't it fail validation here?

Reply:
It would fail using the external create API route, but not using the legacy import API (which does not validate object fields)